ext/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import * as priority from './priority'
+import * as tags from './tags'
+import * as formats from './formats'
+
+export { priority, tags, formats }
```

Comment:
Kinds and types should also be exported for both JS and TS.

Reply:
Done.